.circleci/config.yml

```diff
@@ -537,15 +537,15 @@ jobs:
                 working_directory: ~/workbench/api
                 command: ./project.rb run-local-api-tests
 
-  api-deps-check:
-    executor: workbench-executor
-    steps:
-      - run-api-test:
-          additional_steps:
-            - run:
-                name: Scan dependencies for vulnerabilities
-                working_directory: ~/workbench/api
-                command: ./gradlew dependencyCheckAnalyze --info
+#  api-deps-check:
```

Comment:
need to remove as a dependency from `deploy-staging-and-stable` (in 2 locations)